doc/docs.md

```diff
@@ -20,15 +20,21 @@ The major way to get the latest and greatest V, is to __install it from source__
 It is __easy__, and it usually takes __only a few seconds__.
 
 ### Linux, macOS, FreeBSD, etc:
-You need `git`, and a C compiler like `tcc`, `gcc` or `clang`, and `make`:
+You need [`git`](https://git-scm.com/downloads)
```

Comment:
Do not remove references for the C compiler; on some platforms (apple m1 or freebsd) tcc does not work, or we do not supply prebuild binaries.

Reply:
ahh i see, well that fixes that then.

Reply:
Can you restore the previous sentence?

Reply:
I think I'll just close this since it seems somehow I deleted the fork I made.